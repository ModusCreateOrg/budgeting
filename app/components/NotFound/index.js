import React, { PureComponent } from 'react';
import { PropTypes } from 'prop-types';

export default class NotFound extends PureComponent {
  static propTypes = {
    title: PropTypes.string.isRequired,
    description: PropTypes.string.isRequired,
  };

  render() {
    const { title, description } = this.props;

    return (
      <div>
        <h1>{title}</h1>
        <p>{description}</p>
      </div>
    );
  }
}
