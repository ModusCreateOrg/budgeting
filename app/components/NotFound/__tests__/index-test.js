import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../';

it('renders correctly', () => {
  const tree = renderer.create(<NotFound title="Title" description="Description" />).toJSON();
  expect(tree).toMatchSnapshot();
});
