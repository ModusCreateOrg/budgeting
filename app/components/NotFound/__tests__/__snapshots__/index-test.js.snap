// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Title
  </h1>
  <p>
    Description
  </p>
</div>
`;
